src/main/java/org/jabref/gui/maintable/MainTableHeaderContextMenu.java

```diff
@@ -224,4 +248,35 @@ public boolean isVisibleInTable() {
             return visibleInTable;
         }
     }
+
+    /**
+     * build appropriate actions for the right click menu (different from StandardActions)
+     *
+     */
+    private enum RightClickMenuActions implements Action {
+
+        SHOW_PREFS_RIGHT_CLICK_MENU(Localization.lang("More options..."), IconTheme.JabRefIcons.PREFERENCES);
+
+        private final String text;
+        private final String description;
+        private final Optional<JabRefIcon> icon;
+        private final Optional<KeyBinding> keyBinding;
+
+        RightClickMenuActions(String text, IconTheme.JabRefIcons icon) {
+            this.text = text;
+            this.description = "";
+            this.icon = Optional.of(icon);
+            this.keyBinding = Optional.empty();
+        }
+
+        @Override
+        public String getText() {
+            return text;
+        }
+
+        @Override
+        public Optional<JabRefIcon> getIcon() {
+            return icon;
+        }
+    }
```

Comment:
After some thought: The SHOW_PREFS_RIGHT_CLICK_MENU should move to the StandardActions class and be renamed to SHOW_PREFERENCES, as
- This could be reused in other places.
- From an architectural view: Collect all the Actions of the command pattern in a single place to keep it easier to read and understand.
- Composition over inheritance (?)

Reply:
This is simliar to the StandardActions implementation

Reply:
We had this discussion in the past multiple times and decided against new implementations of the Actions class.

I just introduced a .withText method to provide a solution for this very special occasion when the name must be changed.